engine/src/main/java/org/terasology/config/RenderingConfig.java

```diff
@@ -328,7 +328,7 @@ public void setVignette(boolean vignette) {
     }
 
     public boolean isMotionBlur() {
-        return motionBlur && !isOculusVrSupport();
+        return motionBlur && !isVrSupport();
```

Comment:
I actually ended putting this back in. The reason being, it seems that motionBlur is not being read properly or something - in my settings it's unticked, but blur is active all of the time, and blur makes VR unusable. We can take this out when we fix the blur property reading / whatever the bug is.